src/rules/indentation.js

```diff
@@ -42,7 +47,9 @@ function indentation(parsedFile, unused, configuration) {
 
     scenario.steps.forEach(function(step) {
       // Check Step indentation
-      test(step.location, configuration, 'Step');
+      var keyword = step.keyword.trim();
+      var type = keyword in availableConfigs ? keyword : 'Step';
```

Comment:
What I had in mind is more of a hierarchy/collapsing structure for the configuration. For example let's say i use the following config:
```
{
  "indentation": ["on", {"Step": 3}]
}
```

Given that I haven't specified a configuration for `Given When Then...` I would expect that all the steps are now following my "Step" rule of indentation of 3 spaces/tabs.

If I use:

```
{
  "indentation": ["on", {"Step": 3, "Then":4}]
}
```

All the "Then" steps should have indentation of 4 and the rest should have indentation of 3.

---
If you're interested in adding support for languages outside of english, gherkin(the module) has a mapping for all languages (https://github.com/cucumber/gherkin/blob/master/gherkin-languages.json) which we could use to translate from any language to the english names of the step types. This is kind of what i mean (it doesn't work but I just want to give you and idea of how I think I would implement it):

```
 var availableConfigs = {
...
   'given': 2, // lowercase so they match the keys of the gherkin-languages file
   'when': 2,
...
  };

function indentation(parsedFile, unused, configuration) {
  var languageMapping = require("gherkin/gherkin-languages.json"); 
  var language = languageMapping[parsedFile.language];
  ...
    scenario.steps.forEach(function(step) {
      // Check Step indentation
      var stepType = _.findKey(language, function(o) { return step.keyword.trim() in o;}) || "Step";
      test(step.location, configuration, stepType);
    });
  });
...
}